spinup/algos/pytorch/ppo/ppo.py

```diff
@@ -403,6 +403,28 @@ def update():
             if render:
                 env.render()
 
+            # Our approach below differs significantly from OpenAI's
+            # in several ways worth noting. In their code:
+            # - o is the only var given a temp container and updated post-store
```

Comment:
I would just say variable instead of container here.

Reply:
Also in the csq branch I took a lot of your suggestions and will merge that in eventually. Perhaps it would be good to modify the comment for there as well.